pkg/tfgen/docs.go

```diff
@@ -1331,11 +1331,20 @@ func (g *Generator) convertHCLToString(hcl, path, languageName string) (string,
 		// we write an example to a temp file internally in order to pass to convert.Convert().
 		//
 		// fileName starts with a "/" which is not present in the resulting error, so we need to skip the first rune.
-		errMsg := strings.ReplaceAll(diags.All.Error(), fileName[1:], "")
+		errMsg := ""
+		first := true
+		for _, diag := range diags.All {
+			if !first {
+				errMsg += ", and "
+			}
```

Comment:
Minor bug here: first is never reassigned so this will never add the ", and ".
Can omit it entirely with indexes.

```suggestion
		for idx, diag := range diags.All {
			if idx > 0 {
				errMsg += ", and "
			}
```